packages/collections/persistent/persistent.pony

```diff
@@ -3,5 +3,7 @@
 
 List - A persistent list with functional transformations.
 
-Map - A persistent map based on Bagwell's hash array mapped trie algorithm.
+Map - A persistent map based on the Compressed Hash Array Mapped Prefix-tree
```

Comment:
I copied the benchmarking method from the GitHub repo of rs-persistent-datastructures. If you would want me to benchmark it differently, I can.